### ISSUE TITLE
puglStopTimer() kills more that one timer

### ISSUE THREAD

**Issue by 7890** [lv2/pugl#101]
Hi, this line could be problematic:
https://github.com/lv2/pugl/blob/9b5a0871c1a8771dbe204e60f437653a67abd42a/src/x11.c#L1381
It will eventually remove another registered timer.

```
        memmove(w->timers + i,
                w->timers + i + 1,
                sizeof(PuglTimer) * (w->numTimers - i - 1));
        memset(&w->timers[i], 0, sizeof(PuglTimer)); ///<<< removing this solved weird timer behaviour
```

To trigger the bug, register more than one timer, stop the 1st registered -> 2nd registered will also stop (unexpectedly). Please confirm, 
greetings

**Comment by 7890** [comment]
Let's write a test to detect missing timers, then create a PR with test & fix.

**Comment by drobilla** [comment]
Fixed in 6b31fb5, thanks.
